.eslintrc.js

```diff
@@ -6,22 +6,25 @@ module.exports = {
   },
   extends: [
     'plugin:react/recommended',
-    'standard'
+    'standard-with-typescript'
   ],
   parserOptions: {
     ecmaFeatures: {
       jsx: true
     },
     ecmaVersion: 12,
-    sourceType: 'module'
+    sourceType: 'module',
+    project: './tsconfig.json'
   },
   plugins: [
-    'react'
+    'react',
+    '@typescript-eslint'
   ],
   rules: {
     // proptypes validation as warnings for now
     // in js defining extra prop types is a little
     // cumbersome. Remember to remove if we switch to TS
-    'react/prop-types': 1
+    'react/prop-types': 1,
+    indent: ['warn', 2]
```

Comment:
Is there a way to add these linter errors as warnings rather than stopping the compilation like here?
![image](https://user-images.githubusercontent.com/8848557/104561472-50c2df00-55fc-11eb-9073-62aa92970e82.png)

Reply:
I think formatting errors should stay errors to reduce the chance they accumulate and get committed. Plus if your editor is set up to format on save using the eslint config, then the compilation will never halt for this reason. Where did these come from? Is your editor reformatting based on some other config?

Reply:
it showed up when I ran `npm start`, may be a yarn vs npm thing. I don't have vscode configured to format on save, so I'll add that to my settings.json and that'll probably fix it once I resave the files.